Adding documentation regarding instrumentalized query names. (#321)
newrelic/entity-definitions: docs/golden_metrics.md

### Roll-up entities

> In the cases that the entity type can be ingested from different sources and requires to provide a different query implementation for each of them, you can use `queries` instead of `query`

Reviewer: mmm this probably reads better in its previous state 😬  how about this?
```suggestion
In the cases where the entity type can be ingested from multiple sources, you'll be required to provide a different query implementation per source. In this case, you should use `queries` instead of `query`
```

In this example `prometheus` and `newRelic` are the values the entity must have in the `instrumentation.provider` tag.
The first tag value that matches with the entity will be the one used to build the queries.

```yaml
destinations:
  title: Unique Destinations
  queries:
    kentik/netflow-events:
      select: uniqueCount(dst_addr)
      from: KFlow
      where: "provider = 'kentik-flow-device'"
```

> There's also the possibility to specify both, provider and name in the form of `{provider}/{name}`.

Reviewer: ```suggestion
There's also the possibility to specify both provider and name in the form of `{provider}/{name}`.
```

> 1. `provider` must have in the `instrumentation.provider` tag. (`kentik` in the example)

Reviewer: Do you mean to say this? 
```suggestion
1.  You must add the provider as a value of the `instrumentation.provider` tag. (eg provider: `kentik`, as in the example)
```

> 2. `name` must have in the `instrumentation.name` tag. (`netflow-events` in the example)

Reviewer: ```suggestion
2.  You must add the name of the provider in the `instrumentation.name` tag. (eg provider name: `netflow-events`, as in the example)
```
